playbooks/common/openshift-cluster/set_infra_launch_facts_tasks.yml

```diff
@@ -0,0 +1,15 @@
+---
+- set_fact: k8s_type=infra
+- set_fact: sub_host_type="{{ type }}"
+- set_fact: number_infra="{{ count }}"
+
+- name: Generate infra  instance names(s)
+  set_fact:
+    scratch_name: "{{ cluster_id }}-{{ k8s_type }}-{{ sub_host_type }}-{{ '%05x' | format(1048576 | random) }}"
+  register: infra_names_output
+  with_sequence: count={{ number_infra }}
+
+- set_fact:
+    infra_names: "{{ infra_names_output.results | default([])
```

Comment:
Per the best practices guide, we should always pass in "true" as the second param of default, unless we're dealing with boolean values.

So, this should be:
```
infra_names: "{{ infra_names_output.results | default([], true)
```

Best practices guide:
https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#filters